src/parse-options.js

```diff
@@ -13,6 +17,13 @@ const DEFAULT_OPTIONS = {
 function parseOptions (providedOptions) {
   let options = Object.assign({}, DEFAULT_OPTIONS, providedOptions)
 
+  // normalizing options.color to be a boolean or a color value
+  options.color = (options.color === 'false' || options.color === false) ? false : options.color
+
+  if (!(options.color === false || options.color in styles)) {
+    throw new Error('Invalid color option. Set color to true or to one of the value from https://github.com/chalk/ansi-styles#colors')
```

Comment:
I don't think we should tell them to set the color to `true`, if they've passed a color it should be one of the values. The default behavior (of not passing `color` or `--color`) will provide color.

How about just `'Invalid color option. Color must be one of the supported chalk colors:  https://github.com/chalk/ansi-styles#colors`